src/Pages/ArticleDetailPage.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+import { css } from "glamor";
+import { observer } from "mobx-react";
+
+import ReactMarkdown from "react-markdown";
+import ArticleHeader from "./../Components/ArticleHeader";
+import SmallCardPost from "./../Components/SmallCardPost";
+
+const ArticlePageObserver = observer(
+  class ArticlePageObserver extends Component {
+    componentWillReact() {
```

Comment:
This method is new to me!

Reply:
this is a lifecycle hook added by mobx-react. this will fire when there is a change in observables.